test/distributed/org/apache/cassandra/distributed/test/cql3/SingleNodeTableWalkTest.java

```diff
@@ -424,7 +448,8 @@ public State(RandomSource rs, Cluster cluster)
                                                                   .withoutTransaction()
                                                                   .withoutTtl()
                                                                   .withoutTimestamp()
-                                                                  .withPartitions(SourceDSL.arbitrary().pick(uniquePartitions));
+                                                                  .withPartitions(SourceDSL.arbitrary().pick(uniquePartitions))
+                                                                  .withColumnExpressions(e -> e.withOperators(Generators.fromGen(BOOLEAN_DISTRIBUTION.next(rs))));
```

Comment:
each column will have a different distribution for when operators are allowed

Reply:
Required for CASSANDRA-20449 repro at least 😄

Reply:
wasn't required; we hit the issue before this change.  This change is to better cover different edge cases

Reply:
We need different assignment operators to repro (`+= v` vs. `= {v}`) but not necessarily different distributions of operators, which is what's introduced here - is that right?